docs/oauth2-examples-multiresource.md

```diff
@@ -0,0 +1,174 @@
+      ]
+    }
+  },
+  "scope": "profile email rabbitmq.read:*/* rabbitmq.write:*/* rabbitmq.configure:*/*",
+  "clientId": "prod_producer",
+  "clientHost": "172.18.0.1",
+  "email_verified": false,
+  "preferred_username": "service-account-prod_producer",
+  "clientAddress": "172.18.0.1"
+}
+</pre>
+</li>
+<li>Similarly, launch AMQP producer <code>dev_producer</code>, registered in Keycloak too but with the permission to access <code>rabbit_dev</code> resource:
+<pre class="lang-bash">
+make start-perftest-producer-with-token PRODUCER=dev_producer TOKEN=$(bin/keycloak/token dev_producer z1PNm47wfWyulTnAaDOf1AggTy3MxX2H)
+```
+
+### Test Management UI accessed via two separate resources
+
+This is a summary of the configuration, found in [rabbitmq.conf](https://github.com/rabbitmq/rabbitmq-oauth2-tutorial/tree/support-multiple-resource-server-ids/conf/multi-keycloak/rabbitmq.conf):
+
+* There are two users declared in Keycloak: `prod_user` and `dev_user`
+* The two resources, `rabbit_prod` and `rabbit_dev`, are declared in the Rabbitmq management plugin with their own OAuth2 client (`rabbit_prod_mgt_ui` and `rabbit_dev_mgt_ui`), scopes and the label associated to each resource.
+```
+management.oauth_resource_servers.1.id = rabbit_prod
+management.oauth_resource_servers.1.client_id = rabbit_prod_mgt_ui
+management.oauth_resource_servers.1.label = RabbitMQ Production
+management.oauth_resource_servers.1.scopes = openid profile rabbitmq.tag:administrator
+
+management.oauth_resource_servers.2.id = rabbit_dev
+management.oauth_resource_servers.2.client_id = rabbit_dev_mgt_ui
+management.oauth_resource_servers.2.label = RabbitMQ Development
+management.oauth_resource_servers.2.scopes = openid profile rabbitmq.tag:management
+```
+* Because there is only one OAuth2 server, both resources share the same `provider_url`:
+<pre class="lang-ini">
+management.oauth_provider_url = http://0.0.0.0:8081/realms/test
+</pre>
+* Each OAuth2 client, `rabbit_prod_mgt_ui` and `rabbit_dev_mgt_ui`, has been declared in Keycloak so that they can only emit tokens for their respective audience, be it `rabbit_prod` and `rabbit_dev` respectively.
+
+Follow the steps:
+
+1. Go to the [Management UI](http://localhost:15672)
+1. Select `RabbitMQ Production` resource>
+1. Login as `prod_user`:`prod_user`
+1. Keycloak prompts you to authorize various scopes for `prod_user`
+1. You should now get redirected to the Management UI as `prod_user` user
+
+Now, logout and repeat the same steps for `dev_user` user. For this user, RabbitMQ has been configured to request only `rabbitmq.tag:management` scope.
+
+**Note**: If on step 3, you login as `dev_user`, RabbitMQ will not authorize the user because RabbitMQ has been configured to request the scope `rabbitmq.tag:administrator` for `RabbitMQ Production` however the `dev_user` does not have that scope, but `rabbitmq.tag:management`. Therefore, the user gets a token which has none of the scopes RabbitMQ supports.
```

Comment:
Rewrite: **Note**: In step 3, if you login as the `dev_user`, RabbitMQ will not authorize the `dev_user` because RabbitMQ is configured to request the scope: `rabbitmq.tag:administrator` for `RabbitMQ Production`. The `dev_user` does not have the `rabbitmq.tag:administrator`  scope, it has the `rabbitmq.tag:management` scope. In this scenario, the `dev_user` gets a token which has none of the scopes RabbitMQ supports.

Please check if I still kept the point of this note with the rewrite above, if I have please use it thanks.